packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testOpenPreferences.st

```diff
@@ -0,0 +1,16 @@
+as yet unclassified
+testOpenPreferences
+
+	| prefBrowser prefBrowserMorph |
+
+	klass new openPreferences.
+
+	prefBrowserMorph := [World submorphs detect: [ :morph | morph class name = #PreferenceBrowserMorph]]
+		on: NotFound
+		do: [:exc | Transcript show: exc messageText; cr. ^nil].
```

Comment:
I though this return only works for the [] block like it's the value passed to the variable 'prefBrowserMorph' so I can check if the variable is nil or not. Is it just 'nil' instead of '^nil' ? I'll fix it tomorrow at pauls device :P